BREAKING.md

```diff
@@ -61,3 +61,7 @@ When preparing a release that can include breaking changes, consider applying ch
     - Owner: @timoffex
     - Deprecated after 0.19.8 (https://github.com/wandb/wandb/pull/9557)
     - Can do after September 2025
+
+- Remove storing system settings in temporary directory
```

Comment:
Should this say "Don't store system settings in temporary directory"? I'm also not entirely sure what it refers to based on just this text; if I was the release manager, I wouldn't know how to make it happen.